index.html

```diff
@@ -36,7 +35,6 @@
         <div class="content">
 
          
-            <!-- a video starts -->
               <div class="video">
                  <div class="thumbnail">
                   <iframe width="685" height="415" src="https://www.youtube.com/embed/lJIrF4YjHfQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
I suggest you use semantic HTML tags (ie;< section class="video"> ) instead of divs(<div class="video">), as it's the best practice and its required in this project's requirement.